src/serverless-openapi-typescript.ts

```diff
@@ -1,274 +1,302 @@
-import type Serverless from "serverless";
-import fs from "fs";
-import yaml from "js-yaml";
-import {SchemaGenerator, createGenerator} from "ts-json-schema-generator";
-import {upperFirst, camelCase, mergeWith, set, isArray, get, isEmpty, kebabCase} from "lodash" ;
-import {ApiGatewayEvent} from "serverless/plugins/aws/package/compile/events/apiGateway/lib/validate";
-import { mapKeysDeep, mapValuesDeep} from 'deepdash/standalone'
```

Comment:
@eranelbaz please clear this PR from all style changes or changes not related to the proposed feature